src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -157,6 +159,16 @@ void Main(string[] args)
             VisualStudio.Editor.SendKeys("<s");
             VisualStudio.Editor.Verify.CompletionItemsExist("see", "seealso", "summary");
 
+            var completionItems = VisualStudio.Editor.GetCompletionItems();
+            var targetIndex = Array.IndexOf(completionItems, "see");
+            var currentIndex = Array.IndexOf(completionItems, VisualStudio.Editor.GetCurrentCompletionItem());
+            if (currentIndex != targetIndex)
+            {
+                var key = currentIndex < targetIndex ? VirtualKey.Down : VirtualKey.Up;
+                var keys = Enumerable.Repeat(key, Math.Abs(currentIndex - targetIndex)).Cast<object>().ToArray();
+                VisualStudio.Editor.SendKeys(keys);
```

Comment:
📝 Prior to our use of async completion, the recently-used items list was document-specific. Starting with async completion, the list is global. This is a major change in behavior and I'm not confident that everyone will consider it better.

Reply:
➡️ #33824